src/quote/indexQuoteProvider.ts

```diff
@@ -122,6 +131,48 @@ export class FlashMintQuoteProvider
           tx,
         }
       }
+      case FlashMintContractType.leveragedExtended: {
+        if (!zeroExApiV1) {
+          throw new Error('Contract type requires ZeroExApiV1 to be defined')
+        }
+        const leverageExtendedQuoteProvider =
+          new LeveragedExtendedQuoteProvider(provider, zeroExApiV1)
+        const leveragedExtendedQuote =
+          await leverageExtendedQuoteProvider.getQuote(request)
+        if (!leveragedExtendedQuote) return null
+        const builder = new LeveragedExtendedTransactionBuilder(provider)
+        const txRequest: FlashMintLeveragedExtendedBuildRequest = {
+          isMinting,
+          inputToken: inputToken.address,
+          inputTokenSymbol: inputToken.symbol,
+          outputToken: outputToken.address,
+          outputTokenSymbol: outputToken.symbol,
+          inputTokenAmount: BigNumber.from(0), // TODO: leveragedExtendedQuote.inputTokenAmount,
+          outputTokenAmount: BigNumber.from(0), // TODO: leveragedExtendedQuote.outputTokenAmount,
+          swapDataDebtCollateral: leveragedExtendedQuote.swapDataDebtCollateral,
+          swapDataInputOutputToken: leveragedExtendedQuote.swapDataPaymentToken,
+          swapDataInputTokenForETH:
+            leveragedExtendedQuote.swapDataDebtCollateral, // FIXME: add swapDataInputTokenForETH (returned from quote provider)
+          // FIXME: check if new arguments should be set here in the SDK or rather passed as arguments to they can be set from the app
+          priceEstimateInflator: wei(0.9),
+          maxDust: wei(0.00001),
```

Comment:
Also for these values please check with @ckoopmann how to best approach.

I'd say if it's possible to determine values here during testing that are good in general. I'd just config these values here for now. Otherwise if it's more complex, this should probably be part of the `LeveragedExtendedQuoteProvider` and so returned as part of the quote.